spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -102,6 +101,8 @@ enum ChannelState {
   TRYOPEN,
   OPEN,
   CLOSED,
+  FLUSHING,
+  FLUSHINGCOMPLETE,
```

Comment:
Just thought during the call: maybe we could name this states `UPGRADING` and `UPGRADED`?